.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -30,7 +30,7 @@ inputs:
     default: "1.29"
   eksctl_version:
     description: "Version of eksctl to install"
-    default: v0.169.0
+    default: v0.175.0
```

Comment:
Can we create an issue for auto-updating this through automation?

Reply:
Created the issue: https://github.com/aws/karpenter-provider-aws/issues/6096